server/submission/resolvers.js

```diff
@@ -65,15 +110,24 @@ const resolvers = {
       manuscript.id = await db.save(manuscriptDb)
       return manuscript
     },
-    async updateSubmission(_, { data }, ctx) {
-      const manuscript = await db.selectId(data.id)
+
+    async updateSubmission(_, { data, isAutoSave }, ctx) {
+      let manuscript = await db.selectId(data.id)
```

Comment:
Do you need `let` here? `manuscript` (the object) won't change with `lodash.merge`, but the value will (e.g. the props on it).

Reply:
See line 124.

Reply:
Ok